.github/workflows/release.yml

```diff
@@ -19,6 +19,9 @@ jobs:
       with:
         ruby-version: 2.6
     
+    - name: Update bundler
```

Comment:
This ones on me 😅 `actions/setup-ruby` is deprecated in favour or `ruby/setup-ruby`. I believe the action above should use 
```
ruby/setup-ruby@v1
      with:
        ruby-version: 2.6
        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
```
and that should avoid you needing to run a separate action to install bundler.